docs/style.css

```diff
@@ -255,16 +252,16 @@ li {
 	border-width: 0;
 }
 
-#cgit > .content {
-	margin: 1rem 0;
-}
-
 #cgit > .footer {
 	display: flex;
 	justify-content: center;
 	color: #ccc;
 }
 
+#cgit .footer a:first-child {
```

Comment:
☢️ WARNING: NUCLEAR LEVELS OF COZY ☢️

<img width="365" alt="Screenshot 2024-01-25 at 23 19 53" src="https://github.com/underground-software/orbit/assets/1909841/ffc48df3-f343-45ed-83d7-d2ca1672106e">